### PR TITLE
PostgreSql Flexible Server

### PR DESCRIPTION
**Pull request by ppotturi**

### CODE REVIEW

File: .azuredevops/test.yaml
Comment (ajaysoni-cap) [inline .azuredevops/test.yaml:1]: Needs to be deleted...

Comment (ppotturi) [reply .azuredevops/test.yaml:1]: Done

File: .azuredevops/deploy-platform-core.yaml [hunk 1]
```diff
@@ -110,3 +110,6 @@ extends:
             privateEndpointDnsRecordsForResources:
               - resourceName: $(infraResourceNamePrefix)$(nc_resource_containerregistry)$(nc_instance_regionid)01
                 resourceGroupName: $(servicesResourceGroup)
+          - name: flexible-server
+            path: infra/postgre-sql
+            resourceGroupName: $(servicesResourceGroup)
```
Comment (DaRoszko) [inline .azuredevops/deploy-platform-core.yaml:115]: should this be within the 'dbs' resource group for services?

Comment (ppotturi) [reply .azuredevops/deploy-platform-core.yaml:115]: yes, I will change it to dbs

Comment (ppotturi) [reply .azuredevops/deploy-platform-core.yaml]: Done

**Review by ajaysoni-cap — APPROVED** [review]

**Pull request merged**